library/spec/pivotal-ui-react/toggle/toggle_spec.js

```diff
@@ -70,4 +70,35 @@ describe('Toggle', () => {
       expect('.toggle-switch').toBeChecked();
     });
   });
+
+  describe('size attribute', function() {
+    beforeEach(function() {
+      ReactDOM.unmountComponentAtNode(root);
+    });
+
+    it('renders with size=medium by default', function() {
+      ReactDOM.render(<Toggle id='boggle' />, root);
+      expect($('label').attr('class')).toContain('medium');
```

Comment:
@charleshansen Is this something you feel strong about, or only mild? I think it's somewhat easier to read the test with mounting directly about the expectation (I don't have to jump around trying to find out what subject is, and so on). Thoughts?